runtime/rust/python-wheel/examples/bls/bar.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
+
+import uvloop
+from nova_distributed import DistributedRuntime, nova_worker
```

Comment:
will probably need a global replace
```suggestion
from triton_distributed import DistributedRuntime, nova_worker
```

Reply:
updated all `nova_` and `nova-`